ajax/libs/sinon.js/package.json

```diff
@@ -16,6 +16,15 @@
     "type": "git",
     "url": "https://github.com/sinonjs/sinon"
   },
+  "npmName": "sinon",
+  "npmFileMap": [
+    {
+      "basePath": "pkg",
+      "files": [
+        "**/+(sinon.|sinon-no-sourcemaps.)*"
```

Comment:
@fatso83 Sorry to bother you. I got your point about the produced files after 2.0. I still have one question about older versions. Could you please tell me so do we need those no longer produced files (e.g sinon-ie.js and sinon-timers.js) in 1.16.x and 1.17.x to let the library well in 1.16.x and 1.17.x? Thank you!